src/Metadata/Storage/TableMetadataStorage.php

```diff
@@ -240,7 +265,11 @@ private function getExpectedTable(): Table
         $schemaChangelog->addColumn($this->configuration->getExecutedAtColumnName(), 'datetime', ['notnull' => false]);
         $schemaChangelog->addColumn($this->configuration->getExecutionTimeColumnName(), 'integer', ['notnull' => false]);
 
-        $schemaChangelog->setPrimaryKey([$this->configuration->getVersionColumnName()]);
+        if (class_exists(PrimaryKeyConstraint::class)) {
+            $schemaChangelog->addPrimaryKeyConstraint(new PrimaryKeyConstraint(null, [new UnqualifiedName(Identifier::unquoted($this->configuration->getVersionColumnName()))], true));
```

Comment:
The `PrimaryKeyConstraint` constructor is internal. Please use `PrimaryKeyConstraint::editor()` instead. You can also use `UnqualifiedName::unquoted()` instead of the `UnqualifiedName` constructor and `Identifier::unquoted()`.

Reply:
Much simpler than where I copied this stuff from, thanks!